src/update_roster.R

```diff
@@ -259,16 +260,16 @@ build_rosters_weekly_ngsapi <- function(season){
           game_type %in% c("WC", "DIV", "CON", "SB") ~ week + max(week[game_type == "REG"]),
           T ~ week
         ),
-
       )
-    weekly_rosters[["birth_date"]] <- NULL # sometimes ngsscrapR::scrape_rosters() returns this, sometimes it doesn't
+    weekly_rosters[["birth_date"]] <-
```

Comment:
Why blanking birthdates here? that makes 2016+ rosters not have birth dates?

Reply:
For some reason `ngsscrapR` will sometimes return a `birth_date` column and sometimes will not. I just delete this column and pull `birth_date` from shield:

```r
nflreadr::load_rosters(2016) |>
    dplyr::pull(birth_date) |>
    head()
#> [1] "1985-03-22" "1986-12-11" "1990-07-11" "1990-06-17" "1991-09-01"
#> [6] "1992-05-13"
```

Reply:
hmm. but you don't run shield on players entering league after 2016 then?

Reply:
Sorry, I don't follow -- I build shield rosters for pre-2002, build shield and data exchange rosters for 2002-2015, and build shield and ngs rosters for 2016+. Then I join shield to the data exchange and ngs rosters to get cleaned values for height, weight, birthdate, etc.

Reply:
Oh! I did not read L363. Duh.